src/injected/analyzer-controller.ts

```diff
@@ -95,7 +95,7 @@ export class AnalyzerController {
         });
     }
 
-    private getAnalyzerByIdentifier(key: string) {
+    private getAnalyzerByIdentifier(key: string): IAnalyzer<string> {
```

Comment:
This is a little tricky by I think `<string>` is not the correct type for the `IAnalyzer` return type here.

Reply:
We are returning `IAnalyzer` using `key: string`. That was my thinking for this

Reply:
Updated.